index.js

```diff
@@ -6,6 +6,8 @@ const rimraf = require('rimraf');
 const readline = require('readline');
 const URL = require('url').URL;
 const chalk = require('chalk');  // colorize output
+const open = require('open');    // open URL in default browser
+const isCI = require('is-ci');   // honor CI server limitations
```

Comment:
let's not use `is-ci` package / have CI-specific code

Reply:
won't trying to open a browser on a CI server cause problems though?

Reply:
it likely will just fail and you need to catch error or smth. What happens locally if you pipe stdout to file for example?